plugins/mcreator-localization/help/fr_FR/item/enchantability.md

```diff
@@ -1,11 +1,12 @@
-How common rare enchantments can be enchanted on this item/tool. 
-The higher the enchantability, the better enchantments you will get when enchanting the item/tool.
+Comment des enchantements rares et courants peuvent être enchantés sur cet item/outil. 

```

Comment:
*La chance d'avoir des enchantements rares sur cet item/outil* would be better and "more French".

Reply:
Done.